tests/common/devices/sonic_asic.py

```diff
@@ -560,3 +561,17 @@ def get_bgp_statistic(self, stat):
     def check_bgp_statistic(self, stat, value):
         val = self.get_bgp_statistic(stat)
         return val == value
+
+    def get_def_route_frm_app_db(self, af='ipv4'):
```

Comment:
suggest not using abbreviation, like `get_default_route_from_app_db`

Reply:
fixed in latest commit